tests/FSharp.Compiler.Service.Tests/Symbols.fs

```diff
@@ -787,6 +787,46 @@ type T() =
 
         assertRange (9, 19) (9, 20) mProp
 
+
+    [<Fact>]
```

Comment:
I think `ValReprInfo` also is used for active patterns. Do you think we can add a test to check it:
```fsharp
    let ``Repr info AP`` () =
        let _, checkResults =
            getParseAndCheckResults """
module Module
let (|F|_|) x = None
"""
        let mfv = findSymbolByName "F" checkResults :?> FSharpMemberOrFunctionOrValue
        let param = mfv.CurriedParameterGroups[0][0]
        param.Name.Value |> shouldEqual "x"
```